android/templates/build/ti.constants.gradle

```diff
@@ -7,7 +7,7 @@
 
 project.ext {
 	tiNdkVersion = '26.2.11394342'
-	tiAndroidXAppCompatLibVersion = '1.4.1'
+	tiAndroidXAppCompatLibVersion = '1.6.1'
```

Comment:
Is this required for the change?

Reply:
Yes, as both methods, get and set application languages introduced in 1.6^

Reply:
yes, we need to be in the 1.6.0 versioon: `Use the setApplicationLocales() and getApplicationLocales() methods in Appcompat 1.6.0 or higher` ([source](https://developer.android.com/guide/topics/resources/app-languages#androidx-impl))

1.7.x won't work. I've tested that. Needs more libraries to be updated

Reply:
Roger that.

Reply:
1.6.1 was fine too as it is a patch release in 1.6.x it should be the preferred version.